### DISCUSSION TITLE
How to create new QSO Modes

### DISCUSSION THREAD

**Discussion by thelovebug** [magicbug/Cloudlog#1062] [Q&A] [answered]
I've discovered the magic that is non-OTA amateur radio, like Echolink, Brandmeister, and the new M17.  I'm trying to add these as Modes from the Admin menu, and it seems to have a bit of an attitude.

It took me about 5 attempts to create M17 as a Mode, which I then changed to "Echolink" as that was the one I wanted to log more urgently.  Now I can't create another M17 (or any other type).  Is there a limit to how many I can create (i.e. one?) or am I doing something wrong in how I'm creating them?

I'm on a Magicbug-managed instance of Cloudlog.

73 de M7TLB

**Comment by dg9vh** [comment, marked as answer]
Hi,
it is recommended to use the ADIF-defined modes (here I think "Digital
Voice" would be suitable)... Else, within the admin-menu there is a "modes"
menu, where you can add new modes with their (Adif)-specifications...

73 de Kim
DG9VH

Am Do., 10. Juni 2021 um 18:14 Uhr schrieb Dave Lee <
***@***.***>:

> I've discovered the magic that is non-OTA amateur radio, like Echolink,
> Brandmeister, and the new M17. I'm trying to add these as Modes from the
> Admin menu, and it seems to have a bit of an attitude.
>
> It took me about 5 attempts to create M17 as a Mode, which I then changed
> to "Echolink" as that was the one I wanted to log more urgently. Now I
> can't create another M17 (or any other type). Is there a limit to how many
> I can create (i.e. one?) or am I doing something wrong in how I'm creating
> them?
>
> I'm on a Magicbug-managed instance of Cloudlog.
>
> 73 de M7TLB
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/magicbug/Cloudlog/discussions/1062>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADKN5OX6TZ7EMJ7NY3AFH73TSDQFTANCNFSM46O27HFQ>
> .
>


-- 
Amateurfunk ist das, was uns verbindet! Bleiben wir in Kontakt!
Ortsverband Q03, Neunkirchen

**Reply by thelovebug** [reply]
Thanks Kim @dg9vh - much appreciate the response.

Not only did I miss DIGITALVOICE as a Mode, I also missed that "EchoLink" was an option in the Propogation list.

73 doh M7TLB
